run.py

```diff
@@ -43,6 +44,7 @@ def __init__(self, graphs):
         self.graphs = graphs
         self.tchains = list()
         self.friend_tchains = list()
+        self.rcws = list()
```

Comment:
What means `rcws`?

Reply:
it's a list of instantiations of RDataFrameCutWeight, the helper class in ```utils/_run.py```